commands/fetch.ts

```diff
@@ -38,22 +44,27 @@ exports.handler = async options => {
 
   trackCommandUsage('fetch', { mode: cmsPublishMode }, derivedAccountId);
 
+  const { assetVersion, staging } = options;
   try {
     // Fetch and write file/folder.
     await downloadFileOrFolder(
-      derivedAccountId,
+      derivedAccountId!,
       src,
       resolveLocalPath(dest),
       cmsPublishMode,
-      options
+      {
+        assetVersion:
+          assetVersion !== undefined ? `${assetVersion}` : assetVersion,
```

Comment:
This is `type: number` and optional and it needs to be converted to `string | undefined`